pkg/resolver/related.go

```diff
@@ -22,3 +31,274 @@ type SearchRelatedResult struct {
 // 	klog.Info("TODO: Resolve SearchRelatedResult: Items() - model/related.go")
 // 	return nil
 // }
+func (s *SearchResult) buildRelationsQuery() {
+	var whereDs []exp.Expression
+
+	//The level can be parameterized later, if needed
+	whereDs = append(whereDs, goqu.C("level").Lt(4))       // Add filter to select only upto level 4 relationships
```

Comment:
Using level 4 as default seems high. Is this needed for accessing subscriptions?